### PR TITLE
Small fixes and typos

### PR DESCRIPTION
**Pull request by kinow**
This pull request is separated in a few commits, so that any of them can be edited or dropped if necessary (no hard feelings). I had a couple simple changes left after writing the parsec unit tests, so decided to include a few more typos I had seen before.

i. on typos, happy to delete the commit with typos to make it easier to review if it's better. There's just one important I think, in `option_parsers.py`, part of the output presented to users - I think - has a typo on the word _command_

ii. the only change in `app_gcylc.py` is to delete the last return statement. That code cannot be reached (easier to see by clicking on the _"View"_ button to see the rest of the function).

iii. the only change in `fileparse.py` is the removal of a variable that is never used. Later in the code, another variable with same name is declared (or re-declared?). It seems to be no need in declaring it there...

iv. the changes in the new cylc review HTML files are what appears to me as unbalanced HTML tags... that were either not opened or closed.

### CODE REVIEW

File: lib/cylc/cycling/iso8601.py [hunk 1]
```diff
@@ -978,7 +978,7 @@ def test_advanced_exclusions_partial_datetime4(self):
 
     def test_exclusions_to_string(self):
         init(time_zone='Z')
-        # Chack that exclusions are not included where they should not be.
```
Comment (hjoliver) [inline lib/cylc/cycling/iso8601.py]: He heh, "chack" - some of these errors crack me up :grin:

File: lib/cylc/gui/gscan.py [hunk 2]
```diff
@@ -1002,7 +1002,7 @@ def _update_group_counts(self):
                 # Compat:<=7.5.0
                 group_id = suite_info.get(KEY_GROUP)
             # For the purpose of this method, it is OK to handle both
-            # witheld (None) and unset (empty string) together
```
Comment (hjoliver) [inline lib/cylc/gui/gscan.py]: @kinow - are you a genius-level English-as-a-second-language speller, or is your IDE helping you out here? Just curious...

File: lib/cylc/gui/updater_dot.py [hunk 3]
```diff
@@ -33,8 +33,8 @@
 
 class DotUpdater(threading.Thread):
 
-    RIGHT_ARROW = u'\u25b7'  # Unicode enpty triangle facing right.
-    DOWN_ARROW = u'\u25bd'  # Unicode enpty triangle facing down.
```
Comment (hjoliver) [inline lib/cylc/gui/updater_dot.py]: "entpy" - now that's difficult to say!

File: lib/cylc/prerequisite.py [hunk 4]
```diff
@@ -59,7 +59,7 @@ def __init__(self, point, start_point=None):
         # cylc.cycling.PointBase
         self.start_point = start_point
 
-        # List of cycle point strings that this prerequiste depends on.
```
Comment (hjoliver) [inline lib/cylc/prerequisite.py]: That's definitely one of mine.  I think "prerequisite" but my hands/brain persist in typing "prerequiste".

File: lib/cylc/profiling/profile.py [hunk 5]
```diff
@@ -63,7 +63,7 @@ def __str__(self):
 
 class ProfilingKilledException(SuiteFailedException):
     """Exception to handle the event that a user has canceled profiling whilst
-    a suite is runnnig."""
```
Comment (hjoliver) [inline lib/cylc/profiling/profile.py]: Reminds me, Bob Dylan originally went by the name "Elston Gunnn" - with 3 n's ... brilliant!

File: lib/cylc/gui/gscan.py (hunk 2 above)
Comment (kinow) [reply lib/cylc/gui/gscan.py]: Definitely not genius. Most typos were found with PyCharm or mwic (interesting little python project... groups typos by number of occurrences.... showing less frequent at the top).

Some were found when I was reading the code and trying to write tests, or just following the logic in the code. Many comments saved me minutes of debugging and trial and error coding!

Comment (hjoliver) [reply lib/cylc/gui/gscan.py]: Great ... I tend to miss subtle-ish mistakes like "witheld" when reading at speed. :grimacing:

File: lib/cylc/profiling/profile.py (hunk 5 above)
Comment (kinow) [reply lib/cylc/profiling/profile.py]: TIL!

**Pull request merged**